tableaudocumentapi/xfile.py

```diff
@@ -16,10 +38,10 @@ def temporary_directory(*args, **kwargs):
         shutil.rmtree(d)
 
 
-def find_file_in_zip(zip):
-    for filename in zip.namelist():
+def find_file_in_zip(zip_file):
```

Comment:
This change is because `zip` is a built-in so we should not use that as a name of the argument.